src/algorithms/semihdp_sampler.cc

```diff
@@ -699,7 +699,7 @@ void SemiHdpSampler::check() {
     auto it = std::upper_bound(v_sorted.begin(), v_sorted.end(), -1);
     if (it != table_to_private[i].end()) {
       int min_v = *it;
-      assert(min_v == 0);
+      assert(min_v == 0);  // TODO
```

Comment:
These are rightful `assert`'s,  it just checks that the algorithm is running fine in a debug situation